src/ixdat/readers/zilien.py

```diff
@@ -87,6 +94,17 @@ def __init__(self):
         self._cls = None
         self._measurement = None
 
+        # start time of the Zilien measurement
+        self._timestamp = None
+        # a dictionary with metadata general information about the Zilien measurement
+        self._metadata = None
+        # a list with the Zilien TSV series headers
+        self._series_headers = None
+        # a list with the Zilien TSV columns headers
```

Comment:
Sorry to be annoying, but can you add just a bit more here? The names `_series_headers` and `_column_headers` really threw me off. An example in the comment would help: 
```
        # a list with the Zilien TSV series headers, such as "Ionguage value" (see module docstring)
        self._series_headers = None
        # a list with the Zilien TSV columns headers, such as "Time [s]" and "Pressure [mbar]"
        self._column_headers = None
        # a numpy array of the parsed Zilien data, which is a big rectangle with Nan filling
        self._data = None
```